src/PKSim.Presentation/Presenters/Charts/EditTimeProfileAnalysisChartPresenter.cs

```diff
@@ -180,10 +239,34 @@ private void showAnalysis()
          _timeProfileAnalysisChartView.ShowPKAnalysisView();
       }
 
+      private bool populationDataCollectorSupportsDifferentAggregations(IPopulationDataCollector populationDataCollector)
+      {
+         return populationDataCollector is PopulationSimulation;
+      }
+
+      private IEnumerable<QuantityPKParameter> extractPKParameters(PopulationSimulation populationSimulation)
+      {
+         if (populationSimulation == null)
+            return Enumerable.Empty<QuantityPKParameter>();
+         var fields = PopulationAnalysisChart.PopulationAnalysis.AllFields.OfType<PopulationAnalysisOutputField>().Select(x => x.QuantityPath);
+         return fields.SelectMany(x => populationSimulation.PKAnalyses.AllPKParametersFor(x));
+      }
+
       private void calculatePKAnalysis()
       {
          var chartData = CreateChartData();
-         _pkAnalysisPresenter.CalculatePKAnalysis(PopulationDataCollector, chartData);
+         _pkAnalysisPresenter.CalculatePKAnalysisOnCurves(PopulationDataCollector, chartData);
+         _pkAnalysisPresenter.PKAnalysisOnIndividualsEnabled = false;
+         if (!populationDataCollectorSupportsDifferentAggregations(PopulationDataCollector))
+            return;
```

Comment:
Empty line after return . Can you run resharper? (hopefully this does it for you)